src/components/PositionedOverlay/tests/PositionedOverlay.test.tsx

```diff
@@ -100,6 +101,26 @@ describe('<PositionedOverlay />', () => {
     });
   });
 
+  describe('preventInteraction', () => {
+    it('dismisses Tooltip once pointer is no longer over children, when preventInteraction is true', () => {
```

Comment:
The `PositionedOverlay` component doesn't know anything about `Tooltip`. This test is just about passing a boolean prop and measuring the effect it has on the component - which in this case, is appending a class name. We should revise the test description to more accurately describe what is happening.